.github/workflows/build.yaml

```diff
@@ -29,21 +29,27 @@ jobs:
         activate-environment: pyreaddbc
         use-mamba: true
         miniforge-variant: Mambaforge
-    
-    - name: Install dependencies
-      run: |
-        # poetry config experimental.new-installer false
-        poetry config virtualenvs.create false
-        poetry build
-        poetry install
+
+    - name: Install uv
+      uses: astral-sh/setup-uv@v4
+
+    - name: Set up Python
+      run: uv python install
+
+    - name: Install the project
+      run: uv sync --all-extras --dev
+
+    - name: Run tests
+      # For example, using `pytest`
+      run: uv run pytest tests
+
+    - name: Build wheel
+      run: uv build --wheel
 
     - name: Lint with flake8
-      run: |
```

Comment:
This `|` character is needed so the CI can identify the next lines are related to the `run` command

Reply:
nasty syntax!